pairModule/pairingProcesses.py

```diff
@@ -25,58 +26,114 @@ def pairAllTeams():
 
 def pairTeam(team):
     people = getPeople(team)
-    previousPairSets = getPreviousPairs(team)
+    previousPairObjs = getPreviousPairObjs(team)
 
-    wfh_answers = inquirer.prompt([
+    lastPairing = previousPairObjs[0] if len(previousPairObjs) else None
+
+    if lastPairing:
+        print(f'\nLast pairing name: "{lastPairing["name"]}"\n')
+        printPairings(lastPairing["pairings"],
+                      [*lastPairing["unpaired"], *lastPairing["excluded"]])
+
+        redo_answer = inquirer.prompt([
+            inquirer.Confirm(
+                'redo',
+                message='Would you like to ignore and redo the last pairing (displayed above)?',
+                default=False
+            )
+        ])['redo']
+
+        print(
+            "\nThe last pairing will be used to set defaults for the upcoming questions.\n")
```

Comment:
Should this message change based on the redo_answer?

Reply:
I wanted the defaults to be set whether or not people redo the thing. redo in this implementation only means that the previous pairing is ignored in the matching algorithm and it gives you the option of overwriting